### DISCUSSION TITLE
Using roc-recv togetheter with Moodeaudio on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by MatthiasHGW** [roc-streaming/roc-toolkit#673] [Q&A]
Hi and thank you for roc!

I am trying to use roc-recv on a Raspberry Pi running [Moodeaudio](https://[moodeaudio.org](https://moodeaudio.org/)). The problem is that roc-recv claims the ALSA interface even when there is no client connected, so that Moode cannot access the DAC.
Is there a way to have roc-recv detach from the ALSA interface when no client is connected and reclaim it when a client connects? I would like to avoid having to use ssh for manually starting and stopping roc-recv every time.

Thank you - Matthias

**Comment by gavv** [comment]
Hi!

Unfortunately it's not possible now.

I've created a task: https://github.com/roc-streaming/roc-toolkit/issues/680

Can't promise any estimates, but I'll try to address it in upcoming releases.

**Reply by MatthiasHGW** [reply]
Ok, Thank you - I can start and stop roc-recv using shellscripts in the meantime.

When you find the time to implement this, it would be great if the status of roc-recv could be monitored somehow so that Moode won’t try to grab the ALSA interface when it is used by roc.
